metadata.yaml

```diff
@@ -27,6 +27,3 @@ releaseSeries:
   - major: 0
     minor: 4
```

Comment:
I think it would be good to drop non-existing versions from test/e2e/data/shared/**metadata.yaml as well